packages/visual-editor/src/components/contentBlocks/MapboxStaticMap.tsx

```diff
@@ -154,17 +154,19 @@ export const MapboxStaticMapComponent = ({
   const marker = `pin-l+${getPrimaryColor(document)}(${coordinate.longitude},${coordinate.latitude})`;
 
   return (
-    <EntityField
-      displayName={t("coordinate", "Coordinate")}
-      fieldId={coordinateField.field}
-      constantValueEnabled={coordinateField.constantValueEnabled}
-    >
-      <img
-        ref={imgRef}
-        className="components w-full h-full object-cover"
-        src={`https://api.mapbox.com/styles/v1/mapbox/${mapStyle}/static/${marker}/${coordinate.longitude},${coordinate.latitude},${zoom}/${grandparentSize.width.toFixed(0)}x${grandparentSize.height.toFixed(0)}?access_token=${apiKey}`}
-      />
-    </EntityField>
+    <div className="max-w-[1280px] mx-auto w-full">
```

Comment:
I don't think this fixes the problem. The api request needs to submit a max width.